changelog.d/20240221_114505_danyal.faheem_add_learner_dashboard_dev_to_ci.md

```diff
@@ -0,0 +1 @@
+[Bugfix] Added the learner-dashboard-dev image to the gitlab ci. (by @Danyal-Faheem)
```

Comment:
nit: remember to prefix all your changelog entries with a dash "- " such that the full changelog is rendered as a bullet point list.

Reply:
Done, thank you for letting me know!